data/mods/TropiCataclysm/monsters/tropical_nether.json

```diff
@@ -194,7 +194,7 @@
   {
     "id": "mon_jellyfish_box",
     "type": "MONSTER",
-    "name": { "str_sp": "sea papalotl" },
+    "name": { "str": "sea papalotl" },
```

Comment:
Wikitionary says the plural form of papalotl is papalomeh.

https://en.wiktionary.org/wiki/papalomeh